README.md

```diff
@@ -39,18 +39,104 @@ composer require yiisoft/rbac-db
 
 ## General usage
 
-```shell
-rbac/db/init
+### Configuring database connection
+
+Configuration depends on a selected driver. Here is an example for PostgreSQL:
+
+```php
+use Yiisoft\Cache\ArrayCache; // Requires https://github.com/yiisoft/cache
+use Yiisoft\Db\Cache\SchemaCache;
+use Yiisoft\Db\Pgsql\Connection;
+use Yiisoft\Db\Pgsql\Driver;
+
+$pdoDriver = new Driver('pgsql:host=127.0.0.1;dbname=yiitest;port=5432', 'user', 'password');
+$pdoDriver->charset('UTF8');
+$connection = Connection(
+    $pdoDriver, 
+    new SchemaCache(
+        new ArrayCache(), // Any other PSR-16 compatible cache can be used.
+    )
+);
 ```
 
-By default, when called repeatedly, the creation of tables will be skipped if they are already exist. The `--force` flag
-can be added to force drop existing tables and recreate them:
+More comprehensive examples can be found at 
+[Yii Database docs](https://github.com/yiisoft/db/blob/master/docs/en/README.md#prerequisites).
 
-```shell
-rbac/db/init --force
+### Working with schema
+
+In order to keep less dependencies, this package doesn't provide any CLI for working with schema. There are multiple 
+options to choose from:
+
+- Use migration tool like [Yii DB Migration](https://github.com/yiisoft/yii-db-migration). Migrations are dumped as 
+plain SQL in `sql/migrations` folder.
+- Without migrations, `SchemaManager` class can be used. An example of CLI command containing it can be found 
+[here](examples/Command/RbacDbInit.php).
+changes manually.
+- Use plain SQL that is actual at the moment of installing `rbac-db` package (located at the root of `sql` folder).
+
+The structure of plain SQL files:
+
+- `pgsql-up` - apply the changes for PostgreSQL driver.
+- `pgsql-down` - revert the changes for PostgreSQL driver.
+
+Plain SQL assumes using default names for all 3 tables (`auth_` prefix is used):
+
+- `auth_item`.
+- `auth_assignment`.
+- `auth_item_child`.
+
+`SchemaManager` allows to customize table names:
+
+```php
+use Yiisoft\Db\Connection\ConnectionInterface;
+use Yiisoft\Rbac\Db\DbSchemaManager;
+
+/** @var ConnectionInterface $database */
+$schemaManager = new DbSchemaManager(
+    itemsTable: 'custom_items',
+    assignmentsTable: 'custom_assignments',
+    database: $database,
+    itemsChildrenTable: 'custom_items_children',
+);
+$schemaManager->ensureTables();
+$schemaManager->ensureNoTables(); // Note: All existing data will be erased.
+```
+
+In case of updating `rbac-db` package - when using migrations, you only need to run migrate command; otherwise - you
```

Comment:
What is "migrate command"?

Reply:
`migrate:up` in case of using https://github.com/yiisoft/yii-db-migration package.

Reply:
Command 'migrate:up' should also use plain sql files to migrate. May be remove this sentence, but after add ability to use sql-files in Yii DB Migrations add full command as example here?

Reply:
OK. I removed this sentence for now.